test/flags-verify.js

```diff
@@ -1,25 +1,30 @@
 'use strict';
 
 var expect = require('expect');
-var runner = require('gulp-test-tools').gulpRunner;
-var eraseTime = require('gulp-test-tools').eraseTime;
+var exec = require('child_process').exec;
 var path = require('path');
 
+var eraseTime = require('./tool/erase-time');
+var cmdSep = require('./tool/cmd-sep');
+
+var baseDir = path.join(__dirname, '..');
+var gulpCmd = 'node ' + path.join(__dirname, '../bin/gulp.js');
+
 describe('flag: --verify', function() {
 
   it('dependencies with invalid dependency', function(done) {
-    runner({ verbose: false })
-      .gulp('--verify invalid-package.json', '--cwd ./test/fixtures/verify/')
-      .run(cb);
+    exec([
+      'cd ' + baseDir + cmdSep,
+      gulpCmd,
+      '--verify invalid-package.json',
```

Comment:
Does this still work? I thought I delete the blacklist file we used to look for

Reply:
It's working for now, though...

Reply:
#245